### PR TITLE
Make mulligan more interesting

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request

imagine the wizard dying 10 minutes in so you have to sit through a fucking 1h45m bloodsucker round. 
this should make mulligan more interesting
# Spriting

# Wiki Documentation

# Changelog

:cl:  
rscadd: Added cult, nuclear, revolution, wizard, zombie to the mulligan list
rscdel: Bloodsuckers, traitorsuckers, heretics, traitorvamp removed from mulligan list.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/zombie/zombie.dm [hunk 1]
```diff
@@ -18,6 +18,7 @@ GLOBAL_LIST_EMPTY(zombies)
 	required_enemies = 3
 	recommended_enemies = 3
 	enemy_minimum_age = 14
```
Comment (Bibby0110) [inline code/game/gamemodes/zombie/zombie.dm]: no

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/zombie/zombie.dm (hunk 1 above)
Comment (Simplehorror) [reply code/game/gamemodes/zombie/zombie.dm]: ok

**Comment by neeshacark** [comment]
i love this, except nukie which i dont think would work very well but yeah mulligans generally suck

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
these should be TRUE/FALSE

**Comment by Simplehorror** [comment]
> these should be TRUE/FALSE

this is how the rest of the code does it

**Comment by JamieD1** [comment]
Unfortunately those gamemodes are not always valid when wizard can spawn.

**Pull request closed without merging**